src/core/common/instance.hpp

```diff
@@ -458,7 +458,7 @@ class Instance : public otInstance
     LinkRaw mLinkRaw;
 #endif // OPENTHREAD_RADIO || OPENTHREAD_ENABLE_RAW_LINK_API
 #if OPENTHREAD_CONFIG_ENABLE_DYNAMIC_LOG_LEVEL
-    otLogLevel mLogLevel;
+    static otLogLevel mLogLevel;
```

Comment:
Is this the only static variable in OT core?

Reply:
I think it's the only one (not counting the `sInstanceRaw` itself). 
How about moving this into `logging.cpp`?